eitprocessing/datahandling/mixins/equality.py

```diff
@@ -14,19 +14,24 @@ class Equivalence:
     """Mixin class that adds an equality and equivalence check."""
 
     # inspired by: https://stackoverflow.com/a/51743960/5170442
-    def __eq__(self, other: Self) -> bool:
+    def __eq__(self, other: object) -> bool:
```

Comment:
why was this change made?

Reply:
mypy (which was somehow installed/activated in my VSCode) objects agains using any type narrowing for an __eq__ method, and that makes sense. `sequence == "test"` is entirely valid and should not raise any typing issues.